explorer/scan/local_scanner.go

```diff
@@ -212,6 +212,7 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstream *up
 			log.Error().Err(err).Msg("unable to connect to asset")
 			continue
 		}
+		asset = runtime.Provider.Connection.Asset // to ensure we get all the information the connect call gave us
```

Comment:
if you look at line https://github.com/mondoohq/cnquery/blob/f04541179ee30d2d3e88a96cdce811346455fc80/explorer/scan/local_scanner.go#L236-L238 there's some commented out code there that was doing this same thing